zebra-consensus/src/chain.rs

```diff
@@ -296,15 +368,16 @@ where
     (
         chain,
         transaction,
-        groth16_download_handle,
+        // TODO: turn this into a struct?
+        vec![groth16_download_handle, state_checkpoint_verify_handle],
```

Comment:
Optional: return `[JoinHandle<()>; 2]` here and de-structure it in the start command.
```suggestion
        [groth16_download_handle, state_checkpoint_verify_handle],
```

Reply:
I'm probably going to work on this tomorrow.

Reply:
Fixed in b4499bd